samples/javascript_typescript/02.echobot-with-counter/src/index.ts

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as restify from 'restify';
+import * as path from 'path';
+import { config } from 'dotenv';
+
+// Import required bot services. See https://aka.ms/bot-services to learn more about the different parts of a bot.
+import { BotFrameworkAdapter, ConversationState, MemoryStorage } from 'botbuilder';
+
+// Import required bot configuration.
+import { BotConfiguration, IEndpointService } from 'botframework-config';
+
+import { EchoBot } from './bot';
+
+// Read botFilePath and botFileSecret from .env file
+// Note: Ensure you have a .env file and include botFilePath and botFileSecret.
+const ENV_FILE = path.join(__dirname, '..', '.env');
+const loadFromEnv = config({path: ENV_FILE});
+
+// Get the .bot file path
+// See https://aka.ms/about-bot-file to learn more about .bot file its use and bot configuration.const BOT_FILE = path.join(__dirname, '..', (process.env.botFilePath || ''));
```

Comment:
Looks like the const for BOT_FILE got duplicated on this comment, please remove.